examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -46,16 +46,39 @@ Example output of newly created Vrf keypair
 }
 ```
 
-**Step 4: Generate data for certificate proof creation and verification**
+**Step 4: Generate data for ceased sidechain withdrawals proofs creation**
 
-Besides of inputs of first Forger box, creation of Sidechain requires data for backward transfer operations, that data could be generated by next command: 
+SDK based sidechains implement a specific mechanism that allows users to withdraw their coins in case sidechain has ceased. 
+Considering its impact on sidechain performance, the ceased sidechain withdrawal (CSW) feature can be disabled.
+In case CSW is not requested, this step can be skipped, otherwise proof specific data could be generated by next command:
 
-`generateCertProofInfo {"seed": "seed", "maxPks": 7, "threshold": 5, "verificationKeyPath": "/tmp/sidechainapp/cert_marlin_snark_vk", "provingKeyPath": "/tmp/sidechainapp/cert_marlin_snark_pk"}`
+`generateCswProofInfo {"withdrawalEpochLen": 900, "verificationKeyPath": "/tmp/sidechainapp/csw_marlin_snark_vk", "provingKeyPath": "/tmp/sidechainapp/csw_marlin_snark_pk"}`
+
+Example of output is:
+```
+{
+    "withdrawalEpochLen": 900,
+    "verificationKey": "02d7160000000000000400000000000000db16000000000000a3a40000000000000c0000000000000001240b188009319f7b6d12739001b47917cad72fde9d0ffaa7dabf7836b46bb4300000014f0c379235a1038e5851edd33a6586a8f714c4ec898ddcd066f7ae43b7a0f317800001a5efe5004f93193ec208703cd2b9b97d4a323ba0f7b7c1454ba435a3de51b130000001a98bfe4439a478c0dddf5e75f504e21f7681a022253a19bc9b8801c59e3aef39800001487a4d0201582fe4d8eafa36fc5d8d65dc9620fdfa204f19e63cd3a69962ec3c800001263ca5ab720ee91d4db9b668a8fb3304453ac58fef70587681c567b1b27fa332800001fa0c2b7b756fab8968631267404b93c76baf329f05be068f1397e728c2a5960e80000121483d74799c0d77720e12a913ebae05014d521b0dea4235263ccd1b5c97a50c000001b3c7a90ddf7f74f4f24013a6ab97f5b791da9bb0dd109c079db6ff8511d2233780000175a25117ecd062b6eff590623ad2ad97f71afb4ba7f1951b4ba97cb49f87ae3200000171543933865335ccc19f7162f34eb43cd4554b2a61ab33c011ea66b60d2cfc1a0000013799cf00e1c8194ea1195f047330f55056d713b15448565e9158cf37a644153e8000"
+}
+
+```
+Note: `withdrawalEpochLen` - expected to be in a range between 100 and 900 and is the same value as the one declared by `sc_create` command below.
+
+
+**Step 5: Generate data for certificate proof creation and verification**
+
+Besides of inputs of first Forger box, creation of Sidechain requires data for backward transfer operations, that data could be generated by next command:
```

Comment:
note: this description flow is quite related to what was in step 3.
I think it has sense to keep it as it was before a step 4, or at least change the text a bit.

Reply:
Ok, I have changed a bit the description. I would prefer to keep CSW proof data generation before Certificate proof data generation because this way CSW is already explained when the user is at the step for the certificate.